doc/man/8/rbd.rst

```diff
@@ -576,7 +576,9 @@ Commands
   details for every mirror-enabled image in the pool or namespace.
 
 :command:`mirror snapshot schedule add` [-p | --pool *pool*] [--namespace *namespace*] [--image *image*] *interval* [*start-time*]
-  Add mirror snapshot schedule.
+  Add mirror snapshot schedule. The interval can be specified in
```

Comment:
It might be better to write ``interval`` in this context, so that the word appears unambiguously as a part of the command listed above and not just as a word emphasized in italics.

Reply:
That, is: backtick backtick interval backtick backtick with no spaces.

Reply:
I suggested `*` to try to differentiate between positional arguments (e.g. `*interval*`) and options (e.g. `--pool`), but as long as there is _some_ highlighting it's all fine with me -- whatever renders better ;)